aiohttp/web_reqrep.py

```diff
@@ -72,6 +77,11 @@ def content_length(self, _CONTENT_LENGTH=hdrs.CONTENT_LENGTH):
 FileField = collections.namedtuple('Field', 'name filename file content_type')
 
 
+class ContentCoding(enum.Enum):
+    deflate = 1
```

Comment:
I believe [this](https://www.iana.org/assignments/http-parameters/http-parameters.xhtml#content-coding) is the current list of registered codings.  I'll add a comment about how this is enum only represents our supported coding types.